projects/storefrontlib/src/cms-structure/page/slot/page-slot.component.html

```diff
@@ -1,6 +1,7 @@
 <ng-template
   [cxOutlet]="position$ | async"
   [cxOutletContext]="{ components$: components$ }"
+  [cxSkip]="position$ | async"
```

Comment:
`cxSkip` is a bit generic, what about `cxSkipLink` (which seems to be THE common term used for this feature)